vnext/Desktop.UnitTests/UnicodeTestStrings.cpp

```diff
@@ -23,7 +23,7 @@ std::vector<std::string> g_utf8TestStrings =
         u8"æ«ââèêøóòôå",
         u8"誨 後竊聽 ﻿白圭志 」 關雎 不稱讚 覽 曰： 第十一回 矣 出 耳. 關雎 」 事 ，可 意 誨 出. 樂而不淫 危德至 ﻿白圭志 後竊聽 分得意. 汗流如雨 吉安而來 父親回衙. 去 矣 覽 曰： 事. 父親回衙 意 玉，不題 汗流如雨 矣 」 覽 誨 去 吉安而來 出 冒認收了. 冒認收了 吉安而來 父親回衙 玉，不題 汗流如雨. 以測機 分得意 建章曰： 在一處. 分得意 建章曰： 第十一回 在一處 樂而不淫 後竊聽. 第十一回 訖乃返 分得意 建章曰：. 意 矣 曰： 耳 誨 去 」 事. 德泉淹 曰： 耳 第七回 招」 出 第五回 ，可 矣 意 」 誨. 父親回衙 汗流如雨 吉安而來 玉，不題 冒認收了. ",
         // A very long string
```

Comment:
>// A very long string [](start = 8, length = 21)

Why is it changed? I assume the intent was to have a very long string, while change makes it shorter.

Reply:
The compiler was emitting a warning saying the string was too long and was being truncated. This fixes the warning, and presumably the truncation matches the edit.

Reply:
Haha.. that's awesome that its supposed to be testing long string handling, and the compiler is truncating the test.